### DISCUSSION TITLE
Welcome to Notas Discussions!

### DISCUSSION THREAD

**Discussion by javierobcn** [javierobcn/Notas#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikeViamTW** [comment]
Thank you for the lines about adding footnotes to mkdocs. I looked everywhere to see how to do that and your page was the only one I found with what to add in the config.

**Reply by javierobcn** [reply]
No problem! Thanks for your message, I'm glad it helped you.
